client/package.json

```diff
@@ -19,7 +19,10 @@
     "postinstall": "npm rebuild node-sass"
   },
   "license": "GPLv3",
-  "dependencies": {},
+  "dependencies": {
+    "ngx-clipboard": "^8.1.3",
```

Comment:
When installing a new dependency, please use `yarn add --dev mydep` so the dependencies go in the `devDependencies` section, and the `yarn.lock` file is updated.

We want the client dependencies to be just for dev because when we'll release peertube, the front-end will be precompiled.